TUM Campus App/TUMPickerController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  TUMPickerController.swift
+//  Campus
+//
+//  Created by Tim Gymnich on 28.06.18.
+//  Copyright © 2018 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+
+protocol TUMPickerControllerDelegateProtocol: class {
```

Comment:
Making this comment here, due to a lack of a better place: In some cases, the UIAlertController expands underneath the status bar. We should try to prevent this.